src/utils/storage.js

```diff
@@ -140,3 +140,16 @@ export const updateUserWordList = async (wordList) => {
     return false;
   }
 };
+
+export const getAllUsers = async () => {
+  try {
+    const allKeys = await AsyncStorage.getAllKeys();
```

Comment:
This strikes me as potentially problematic. `AsyncStorage`, while it is scoped to your app, is global and can contain values set by libraries, etc. (`"Gets all keys known to your app; for all callers, libraries, etc."`)

It looks like you're trying to store a list of users but you're storing them as separate items in `AsyncStorage`. Is that right? Could you store them all in an array under one key?

Reply:
Ah okay I wasn't aware of that. I can do that, but I'll add it to a new PR cause that may take a fair chunk of time to fix